lib/main.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:bricio_vellaz/home.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: ThemeData(
```

Comment:
this theme data can be moved into a new file, to inject the configuration, and add the values into constants when posible